pkg/index/tree.go

```diff
@@ -93,6 +93,18 @@ func postOrderTraversal[V Value](node *Node[V], callback func(node *Node[V], dep
 	callback(node, depth)
 }
 
+// postOrderTraversalAll traverses the whole tree (include tombstones) with postorder traversal.
+func postOrderTraversalAll[V Value](node *Node[V], callback func(node *Node[V], depth int), depth int) {
```

Comment:
```suggestion
func postorderTraversalAll[V Value](node *Node[V], callback func(node *Node[V], depth int), depth int) {
```

Previous codes also did, but I think we could replace `postOrder` with `postorder`.